frontend/src/app.css

```diff
@@ -43,3 +43,14 @@
     columns: 5rem auto;
   }
 }
+
+@variants {
+  .filters-content{
+    height: calc(100% - 110px);
```

Comment:
`110px` is a magic value, which is a bad practice. Moreover, seems that in this case it leads to the following issue: http://prntscr.com/10hyg5g. Please consider another approach for placing filters content below the filters header. Simple `flex-col` will not work?